README.md

```diff
@@ -1,35 +1,51 @@
 # dioxuslabs.com
 
-This repository contains the source code for the https://dioxuslabs.com website. 
+This repository contains the source code for the https://dioxuslabs.com website.
 
-This website is written with Dioxus, pre-generated with `dioxus_ssr`, and then rehydrated with interactivity provided by `dioxus_web`.
+This website is written with Dioxus, pre-generated with `dioxus_ssr`, and then
+rehydrated with interactivity provided by `dioxus_web`.
 
 ## Development
 
-Run the following command in the root of the project to start the tailwind css compiler:
+Run the following command in the root of the project to start the Tailwind CSS
+compiler:
 
-```bash
+```sh
 npx tailwindcss -i ./input.css -o ./public/tailwind.css --watch
 ```
 
-The documentation can be edited using any text editor. Most commonly used editors support syntax highlighting for the `markdown` format. To view your changes you can install the [dioxus-cli] tool locally, assuming you already have a working `Rust` setup;
-```console
+The documentation can be edited using any text editor. Most commonly used
+editors support syntax highlighting for the `markdown` format. To view your
+changes you can install the [`dx`](dx) tool locally, assuming you already have a
```

Comment:
The `[dx](dx)` link leads to nowhere. You can link to the dioxus cli on [github](https://github.com/DioxusLabs/dioxus/tree/master/packages/cli) or [the docs](https://dioxuslabs.com/learn/0.4/CLI)

Reply:
oh, right. I thought I can add `` without changing the `[dx]`-like link.

Oh, I can! https://www.markdownguide.org/basic-syntax/#formatting-the-first-part-of-the-link

Reply:
Yeah, works as I wanted: https://github.com/DioxusLabs/docsite/blob/37014e8b61c15f7bbc55af7bb8becf503c126ea5/README.md.